libraries/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -16,6 +16,7 @@ import {
   Colorize,
   type ITerminal
 } from '@rushstack/terminal';
+import dotenv from 'dotenv';
```

Comment:
Node 20+ is adding a built-in API for this: https://nodejs.org/docs/latest-v20.x/api/util.html#utilparseenvcontent

Reply:
That isn't considered stable yet.